clusters/hive/pools/cvp/cvp-ocp-4-7-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4-7-amd64-aws-us-east-1
+  namespace: cvp-cluster-pool
+  labels:
+    product: ocp
+    version: "4.7"
+    architecture: amd64
+    cloud: aws
+    owner: dpp
+    region: us-east-1
+spec:
+  baseDomain: hive.aws.ci.openshift.org
+  imageSetRef:
+    name: ocp-4.7.0-amd64
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1
```

Comment:
This is a separate resource and should needs to be added too.

Reply:
Resource added, thanks for notifying.